go.mod

```diff
@@ -12,8 +12,8 @@ require (
 	github.com/smacker/go-tree-sitter v0.0.0-20230501083651-a7d92773b3aa
 	github.com/spf13/cobra v0.0.5
 	github.com/stretchr/testify v1.8.1
-	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
-	golang.org/x/net v0.5.0
+	golang.org/x/crypto v0.14.0
```

Comment:
I think we can remove x/crypto after this. It only was used in table.go.

Reply:
Oh, for some reason I thought crypto/md5 comes from that, but it's part of standard lib. Will fix, nice catch.